api/checks/api.go

```diff
@@ -176,3 +179,17 @@ func (s checksAPIImpl) CreateWPTCheckSuite(appID, installationID int64, sha stri
 	}
 	return suite != nil, err
 }
+
+func (s checksAPIImpl) GetWPTRepoAppInstallationIDs() (appID, installationID int64) {
+	aeID := appengine.AppID(s.ctx)
+	domainAndID := strings.Split(aeID, ":")
```

Comment:
Nit: maybe a comment to give an example of `aeID` (I tend to do this whenever I'm doing non-trivial string manipulation).

Reply:
Besides, in the method below, you split on "." instead of ":". Can you confirm?

https://github.com/web-platform-tests/wpt.fyi/blob/bff764ad5ee1df69402e10253bd50a13316167c9/shared/appengine.go#L145

Reply:
Yah, not sure why the inconsistency, but the ID format is documented on the `AppID` method itself.